templates/land-see/kim-myeong.amp.html

```diff
@@ -55,15 +55,147 @@
     </amp-animation>
     {{> ../../components/navbar/navbar.snip.html}}
 
-    <div class="mx-auto m3 max-width-4">
-      <h1 class="pt3">Hello AMP!</h1>
-      <p class="py3">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-      </p>
-      <p class="py3">
-        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?
+    {{#land-see-story-content}}
+    <section class="land-see-story-detail relative mx-auto mb4 pt2">
+      {{#land-see-post-detail}}
+      <div class="mx-auto center {{post-classes}}">
+        <!-- {{#image}}
```

Comment:
remove this?

Reply:
I'm still styling the main hero image as it's unique in its proportions and supposed to swap out for `amp-fx-flying-carpet` in sm breakpoint. This will be reenabled in my next PR, but I'll remove for now.